Runtime/SpriteSwitchMixerBehaviour.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Playables;
+
+namespace Unity.MaterialSwitch
+{
+    public class SpriteSwitchMixerPlayableBehaviour : PlayableBehaviour
+    {
+
+        public override void ProcessFrame(Playable playable, FrameData info, object playerData)
+        {
+            var group = playerData as SelectionGroups.SelectionGroup;
+            if (group == null) return;
+            if (!group.TryGetComponent(out SpriteGroup spriteGroup))
+            {
+                spriteGroup = group.gameObject.AddComponent<SpriteGroup>();
+            }
+            if (Application.isEditor && !Application.isPlaying)
```

Comment:
We are already using `EditorApplication.hierarchyChanged` to collect sprite renderers. Should we do this every frame, even in the editor ?

Reply:
Yes, it matches the behaviour in the MaterialSwitch class. I believe we found cases where hierarchyChanged was not triggered when we needed.

Reply:
I thought we agreed to remove this  section ?